identity/What-We-Own.md

```diff
@@ -1,13 +0,0 @@
-# What repositories we manage
-
-## Current owned services in alphabetical order
-| Name | Description | Name Origin
-| ---- | ----------- | ----------- |
```

Comment:
Until all teams have fully realized their owner ship of the services previously owned by Saturn I think this is valuable to keep around so the context is not lost.

Reply:
Fair enough, this PR had dropped off my radar anyway. I'll close it for now.